Lykke.Job.LykkeJob/src/Lykke.Job.LykkeJob/Program.cs

```diff
@@ -5,18 +5,18 @@
 
 namespace Lykke.Job.LykkeJob
 {
-    public class Program
+    internal sealed class Program
     {
         public static string EnvInfo => Environment.GetEnvironmentVariable("ENV_INFO");
 
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
             Console.WriteLine($"LykkeJob version {Microsoft.Extensions.PlatformAbstractions.PlatformServices.Default.Application.ApplicationVersion}");
-//#$if DEBUG
+            //#$if DEBUG
```

Comment:
These indentions will lead to undesirable preprocessor derictive indentions in the generated project